### DISCUSSION TITLE
Energy Dashboard: Custom Date Range

### DISCUSSION THREAD

**Discussion by kloon** [home-assistant/frontend#17283] [Dashboard]
The energy dashboard is great, however, one feature it lacks is the ability to select a custom date range.

There are options to select Weekly, Monthly, and Yearly, but the problem with this is that it is fixed to specific periods.

Having the ability to select custom date ranges would be very beneficial for comparing data against utility bills which do not start on the first of every month in my country, nor does it always cover a fixed period like a month.

**Comment by flixlix** [comment]
Hey!

I created this custom Card a couple of months ago, that introduced this feature, might be of interest to you :)

https://github.com/flixlix/energy-period-selector-plus
